packages/docs/src/components/Modal/Modal.stories.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
```

Comment:
Please remove all the prop types stuff and export the examples from 'examples/index.js' to match the other docs